### DISCUSSION TITLE
I only have 239000 images, why are there 690000 pending tasks on "GENERATE THUMBNAILS"?

### DISCUSSION THREAD

**Discussion by nodis** [immich-app/immich#4869] [Q&A] [answered]
I only have 239000 images, why are there 690000 pending tasks on "GENERATE THUMBNAILS"?

![微信截图_20231106225032](https://github.com/immich-app/immich/assets/23695589/038f2216-ff5f-449e-a889-97fb90442df7)
![微信截图_20231106225104](https://github.com/immich-app/immich/assets/23695589/48509151-924c-4e9a-89e9-3e7ef33bb84f)

**Comment by alextran1502** [comment, marked as answer]
There are multiple thumbnail jobs within that queue.

- jpeg thumbnail
- webp thumbnail
- face thumbnail
- thumbhash thumbnail
